python/paddle/fluid/tests/unittests/ir/inference/test_trt_conv_pass.py

```diff
@@ -21,6 +21,11 @@
 import paddle.fluid.core as core
 from paddle.fluid.core import PassVersionChecker
 from paddle.fluid.core import AnalysisConfig
+import paddle
+
+if paddle.device.cuda.get_device_name() == "NVIDIA A10":
```

Comment:
Maybe just set `os.environ['NVIDIA_TF32_OVERRIDE'] = '0'`  is ok

Reply:
> Maybe just set `os.environ['NVIDIA_TF32_OVERRIDE'] = '0'` is ok

Done~